code/game/gamemodes/cult/items/armor.dm

```diff
@@ -14,29 +14,29 @@
 	flags_inv = HIDEGLOVES|HIDEJUMPSUIT
 
 /obj/item/clothing/head/helmet/space/cult
-	name = "eldritch helmet"
-	desc = "A bulky helmet, bristling with spikes. It looks space proof."
+	name = "eldritch voidsuit helmet"
+	desc = "A bulky armoured voidsuit helmet, bristling with menacing spikes. It looks space proof."
 	icon_state = "cult_helmet"
-	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
+	armor = list(melee = 60, bullet = 50, laser = 45, energy = 15, bomb = 30, bio = 30, rad = 50)
```

Comment:
Not a fan of these buffs in combination with the reforging, as reforging cult armour to a space suit should have some downside (which is currently in the form of slightly less protection)